packages/main/src/DatePicker.js

```diff
@@ -503,9 +504,8 @@ class DatePicker extends UI5Element {
 	_getTimeStampFromString(value) {
 		const jsDate = this.getFormat().parse(value);
 		if (jsDate) {
-			const jsDateTimeNow = Date.UTC(jsDate.getFullYear(), jsDate.getMonth(), jsDate.getDate());
-			const calDate = CalendarDate.fromTimestamp(jsDateTimeNow, this._primaryCalendarType);
-			return calDate.valueOf();
+			const calDate = CalendarDate.fromLocalJSDate(jsDate, this._primaryCalendarType);
```

Comment:
This is the same as Date.UTC(jsDate.getFullYear(), jsDate.getMonth(), jsDate.getDate());

Reply:
Yes, it is the same. Just in this case the code becomes shorter.